src/chart/model/scale.model.ts

```diff
@@ -231,7 +230,7 @@ export class ScaleModel extends ViewportModel {
 	 * @param fireChanged
 	 * @param forceNoAutoScale - force NOT apply auto-scaling (for lazy loading)
 	 */
-	public setXScale(xStart: Unit, xEnd: Unit, forceNoAnimation: boolean = true) {
+	public setXScale(xStart: Unit, xEnd: Unit, forceNoAnimation: boolean = true, forceUpdateY = true) {
```

Comment:
omg, `forceUpdateY` argument looks like smthing very bad. Please find out how to escape it. 
As you can see there's already two arguments like this, obviously it will grow in numbers if we leave it like that